src/css/catalog/CatalogFilters.css

```
@@ -0,0 +1,24 @@
+#filter-item {
```

Since there are multiple of these, like for `CourseCard` this needs to be `class` instead of `id`.
```suggestion
.filter-item {
```